src/serverdlg.cpp

```diff
@@ -180,6 +170,15 @@ CServerDlg::CServerDlg ( CServer* pNServP, CServerSettings* pNSetP, const bool b
                                          tr ( "Click the button to clear the currently selected recording directory. "
                                               "This will prevent recording until a new value is selected." ) );
 
+    // custom directory
+    QString strCustomDirectory = "<b>" + tr ( "Custom Directory" ) + ":</b> " +
```

Comment:
I agree with your comment arising from the fact that "Directory" is used in two different ways in close proximity (filesystem directory for recording, and directory server for registration). I don't think it needs changing all the way back to "Custom Directory Server Address", but I think "Custom Directory Server" would be clearer. And similarly for the associated What's This and accessibility texts.

Reply:
I prefer the longer for as it's the "Server Address" for a "Custom Directory".  Not a "Custom" "Directory Server".

Reply:
OK, so did your original comment mean you are dealing with it in the next PR rather than this one?

Reply:
Yes.  Remaining after this now looks like:
- feature/server-list-persistence-ui
The main update to get the server list persistence file working, plus amending server registration to lose the checkbox
- patch/language-selection
Small layout change for the Server UI language selector
- patch/recorder-adjustments
Some changes in the layout for the Server Recorder (it was more crowded than needed), plus the labelling change referred to here and getting rid of `QString::null` (as that was all server recorder code).